bazel/com_google_cel_cpp.patch

```diff
@@ -6,7 +6,7 @@ index d99239c..4dc8cae 100644
      return registry->Register(std::move(status.ValueOrDie()));
    }
  
-+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
I meant the opposite, `defined(__clang_major_version__) && __clang_major_version__ >= 8 && !defined(__APPLE__)` for optimized path.

Reply:
Oops, sorry. Fixed.